src/backend/server.ts

```diff
@@ -1,37 +1,34 @@
 import {NextFunction, Request, Response} from 'express';
+import userRoutes from './routes/user';
+import express from 'express';
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+import cors from 'cors';
+import path from 'path';
 
-const express = require("express");
-const mongoose = require("mongoose");
-require("dotenv").config();
+const app = express();
+const PORT = process.env.PORT || 5000;
 
-//Start Express App
-const backendApp = express();
-
-//Connect to MongooseDB
-//TODO: Figure out how to host a local MongoDB or decide to use Atlas MongoDB
-mongoose.connect()
-    .then(() => {
-
-        //Sets the Express app to listen to a specific port
-        const PORT = process.env.PORT;
-        backendApp.listen(PORT, () => {
-            console.log(`Express App running on ${PORT}`);
-        })
+app.use(cors());
+app.use(express.json());
 
-    })
-    .catch((error) => {
-        console.log(`Error connecting mongoose: ${error}`);
-    });
+//Start Express App
+app.get("/", (req: Request, res: Response) => {
+    res.send("Hello World");
+    console.log("base page");
+});
 
-//Pre-handle of every request
-backendApp.use(express.json());
-backendApp.use((request: Request, response: Response, next: NextFunction) => {
-    console.log(`${request.method} from ${request.path}`);
-    next();
+app.listen(PORT, () => {
```

Comment:
move this server bootup at the end of the file for readability.

Reply:
I just updated it.